### DISCUSSION TITLE
Question regarding multisplit device

### DISCUSSION THREAD

**Discussion by vschild** [omerfaruk-aran/esphome_samsung_hvac_bus#22] [General]
I'm going to install a multisplit device with 3 indoor units connected to 1 outdoor unit. Can I connect one ESP to each indoor unit so i can control each device individually? 
Thanks for your work!

**Comment by lanwin** [comment]
The other way around. You only need one ESP witch is connected to the AC bus wire. It did not matter if its connected to one of the indoor unit or directly on the outdoor unit or in between. On this ESP you can then configure all indoor units.

**Reply by lanwin** [reply]
To make it clear. All the indoor units are connected to the same F1, F2 port on the outdoor unit. Or sometimes the indoor units are connected after each other. But they are all on the same wires.

**Reply by vschild** [reply]
Thanks for your reply. I didnt know that!

**Reply by shogsbro** [reply]
Contextually, it is common in industrial control systems to use RS485 in multidrop topology.
